### PR TITLE
[prometheus-snmp-exporter] Add Tom as maintainer

### LINKED ISSUE #4158: [prometheus-snmp-exporter] Extra maintainers
### Describe the bug a clear and concise description of what the bug is.

Hello,

The `prometheus-snmp-exporter` chart is a few versions behind [prometheus/snmp_exporter](https://github.com/prometheus/snmp_exporter) and there are multiple open PRs to update it that haven't been reviewed:

- #4086
- #4043
- #4020
- #3791 

The maintainers are down as @miouge1 and @xiu according to [CODEOWNERS](https://github.com/prometheus-community/helm-charts/blob/main/.github/CODEOWNERS) but there are errors for @miouge1 (suggesting they may not have write access), so it is just @xiu that gets added to PRs.

Are the maintainers open in having any additional maintainers help out? I would be happy to volunteer if so. Thanks!

### What's your helm version?

N/A

### What's your kubectl version?

N/A

### Which chart?

prometheus-snmp-exporter

### What's the chart version?

N/A

### What happened?

_No response_

### What you expected to happen?

_No response_

### How to reproduce it?

_No response_

### Enter the changed values of values.yaml?

_No response_

### Enter the command that you execute and failing/misfunctioning.

N/A

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by walker-tom**
#### What this PR does / why we need it

Adds myself as a maintainer of the promtheus-snmp-exporter Chart as agreed in #4158.

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #4158 

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by xiu — APPROVED** [review]
LGTM @prometheus-community/helm-charts-admins, can you review and merge? Thanks!

**Comment by monotek** [comment]
@SuperQ or @brancz

can you please add @walker-tom to the https://github.com/orgs/prometheus-community/teams/helm-charts-maintainers group?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -45,7 +45,7 @@
 /charts/prometheus-rabbitmq-exporter/ @desaintmartin @iamabhishek-dubey @juanchimienti @monotek
 /charts/prometheus-redis-exporter/ @acondrat @zanhsieh
 /charts/prometheus-smartctl-exporter/ @kfox1111 @zeritti
-/charts/prometheus-snmp-exporter/ @miouge1 @xiu
+/charts/prometheus-snmp-exporter/ @miouge1 @xiu @walker-tom
```
Comment (SuperQ) [inline .github/CODEOWNERS:48]: This needs to be sorted.

```suggestion
/charts/prometheus-snmp-exporter/ @miouge1 @walker-tom @Nexucis 
```

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Comment by SuperQ** [comment]
CC @miouge1

**Review by monotek — APPROVED** [review]

**Review by SuperQ — APPROVED** [review]

**Comment by SuperQ** [comment]
@xiu, can you re-approve as codeowner?

**Review by xiu — APPROVED** [review]

**Pull request merged**